src/layer/vulkan/shader/unaryop.comp

```diff
@@ -3,40 +3,27 @@
 
 #version 450
 
-layout (constant_id = 0) const int op_type = 0;
+layout (constant_id = 0) const uint n = 0;
+layout (constant_id = 1) const int op_type = 0;
```

Comment:
change the order so specialization constant without a push constant fallback has lower id than others